x-pack/plugins/infra/public/pages/metrics/inventory_view/components/survey_kubernetes.tsx

```diff
@@ -9,81 +9,73 @@ import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiGlobalToastList } from '@elast
 import React from 'react';
 import { FormattedMessage } from '@kbn/i18n-react';
 import useLocalStorage from 'react-use/lib/useLocalStorage';
-import { useWaffleOptionsContext } from '../hooks/use_waffle_options';
 
 const KUBERNETES_TOAST_STORAGE_KEY = 'kubernetesToastKey';
 const KUBERNETES_FEEDBACK_LINK = 'https://ela.st/k8s-feedback';
 
 export const SurveyKubernetes = () => {
-  const { nodeType } = useWaffleOptionsContext();
-  const podNodeType: typeof nodeType = 'pod';
-
   const [isToastSeen, setIsToastSeen] = useLocalStorage(KUBERNETES_TOAST_STORAGE_KEY, false);
   const markToastAsSeen = () => setIsToastSeen(true);
 
   return (
     <>
-      {nodeType === podNodeType && (
-        <>
-          <EuiButton
-            href={KUBERNETES_FEEDBACK_LINK}
-            target="_blank"
-            color="warning"
-            iconType="editorComment"
-            data-test-subj="infra-kubernetes-feedback-link"
-          >
-            <FormattedMessage
-              id="xpack.infra.homePage.tellUsWhatYouThinkK8sLink"
-              defaultMessage="Tell us what you think! (K8s)"
-            />
-          </EuiButton>
-          {!isToastSeen && (
-            <EuiGlobalToastList
-              toastLifeTimeMs={Infinity}
-              dismissToast={markToastAsSeen}
-              toasts={[
-                {
-                  id: 'k8s-toast',
-                  title: (
+      <EuiButton
+        href={KUBERNETES_FEEDBACK_LINK}
+        target="_blank"
+        color="warning"
+        iconType="editorComment"
+        data-test-subj="infra-kubernetes-feedback-link"
+      >
+        <FormattedMessage
+          id="xpack.infra.homePage.tellUsWhatYouThinkK8sLink"
+          defaultMessage="Tell us what you think! (K8s)"
+        />
+      </EuiButton>
+      {!isToastSeen && (
+        <EuiGlobalToastList
+          toastLifeTimeMs={Infinity}
+          dismissToast={markToastAsSeen}
+          toasts={[
+            {
+              id: 'k8s-toast',
+              title: (
+                <FormattedMessage
+                  id="xpack.infra.homePage.kubernetesToastTitle"
+                  defaultMessage="We need your help!"
+                />
+              ),
+              color: 'primary',
+              iconType: 'help',
+              toastLifeTimeMs: 0x7fffffff, // Biggest possible lifetime because we control when it should be visible using isToastSeen
```

Comment:
Could we use `Infinity` here too? Will it do the same? Also, do we need to define the `toastLifeTimeMs` 2 times  (in the `EuiGlobalToastList` and here)
```suggestion
              toastLifeTimeMs: Infinity, // Biggest possible lifetime because we control when it should be visible using isToastSeen
```

Reply:
It is mandatory in `EuiGlobalToastList` so I think we can remove it from here ( I don't remember the reason to have it here as well tbh. Maybe it was set before we added the `EuiGlobalToastList` ) From what I tested it worked without it.